fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/FedoraResourceImpl.java

```diff
@@ -720,6 +681,31 @@ public Node getNodeVersion(final String label) {
 
     }
 
+    /**
+     * Helps ensure that there are no terminating slashes in the predicate.
+     * A terminating slash means ModeShape has trouble extracting the localName, e.g., for
```

Comment:
That was my thought or inclination too when I made the original regex PR, a temporary workaround (in my mind, at least), until we could file a ticket with ModeShape.

So the PR by @acoburn could be used temporarily until ModeShape changes this. I can file a request or submit a PR to them.